redaxo/src/addons/metainfo/lib/handler/handler.php

```diff
@@ -35,6 +35,7 @@ public function renderMetaFields(rex_sql $sqlFields, array $epParams)
 
             $name = (string) $sqlFields->getValue('name');
             $title = $sqlFields->getValue('title');
+            /** @psalm-taint-escape sql */ // it is intended that admins can paste sql queries inside this field
             $params = $sqlFields->getValue('params');
```

Comment:
Teilweise muss man dann halt wie hier, explizit markieren, dass hier unsicherer Content erlaubt ist.
Also der Admin darf in das Feld eine SQL-Query einfügen, und die führen wir weiter unten entsprechend unescaped aus.